moviepy/video/io/html_tools.py

```diff
@@ -73,7 +73,7 @@ def html_embed(
 
     Examples
     =========
-
+    TODO Update example
```

Comment:
Why not to place here "TODO Create example" removing the example and update the example of `ipython_display` function of the same module with:

```python
>>> from moviepy.editor as *
    >>> # later ...
    >>> clip.ipython_display(width=360)
    >>> clip.audio.ipython_display()

    >>> clip.write_gif("test.gif")
    >>> ipython_display('test.gif')

    >>> clip.save_frame("first_frame.jpeg")
    >>> ipython_display("first_frame.jpeg")
```